internal/util-logging/src/main/scala/sbt/internal/util/Terminal.scala

```diff
@@ -308,7 +308,7 @@ object Terminal {
   }
   private[sbt] lazy val isAnsiSupported: Boolean = logFormatEnabled.getOrElse(useColorDefault)
 
-  private[this] val isDumb = "dumb" == System.getenv("TERM")
+  private[this] val isDumb = "dumb" == sys.env("TERM")
```

Comment:
```suggestion
  private[this] val isDumb = Some("dumb") == sys.env.get("TERM")
```

Looks like this behavior changed since  `System.getenv` returns `null` here:
> the string value of the variable, or null if the variable is not defined in the system environment

```
[info]   Cause: java.util.NoSuchElementException: key not found: TERM
[info]   at scala.collection.MapLike.default(MapLike.scala:236)
[info]   at scala.collection.MapLike.default$(MapLike.scala:235)
[info]   at scala.collection.AbstractMap.default(Map.scala:65)
[info]   at scala.collection.MapLike.apply(MapLike.scala:144)
[info]   at scala.collection.MapLike.apply$(MapLike.scala:143)
[info]   at scala.collection.AbstractMap.apply(Map.scala:65)
[info]   at sbt.internal.util.Terminal$.<init>(Terminal.scala:311)
[info]   at sbt.internal.util.Terminal$.<clinit>(Terminal.scala)
[info]   at sbt.internal.util.ConsoleAppender$.apply(ConsoleAppender.scala:208)
[info]   at sbt.internal.util.ConsoleAppender$.apply(ConsoleAppender.scala:199)
```